Packages/ConfCore/ConfCore/BookmarkSyncObject.swift

```diff
@@ -50,10 +50,7 @@ extension Bookmark: SyncObjectConvertible, BelongsToSession {
             do {
                 bookmark.snapshot = try Data(contentsOf: snapshotURL)
             } catch {
-                os_log("Failed to load bookmark snapshot from CloudKit: %{public}@",
-                       log: .default,
-                       type: .fault,
-                       String(describing: error))
+                Logger.default.fault("Failed to load bookmark snapshot from CloudKit: \(String(describing: error), privacy: .public)")
```

Comment:
Should we introduce a custom `Logger` instance for these? I don't like using `Logger.default` these days because it doesn't really help much when you're trying to find a specific message in the logs.

Reply:
We can. I was trying to stick with the old rule of "during a refactor, don't fix things unless you want to break things". But it's probababably ok in this case (famous last words, lol)

Reply:
I'm taking a look at how this all might go a little further.